pysteps/nowcasts/steps.py

```diff
@@ -33,6 +33,863 @@
     DASK_IMPORTED = False
 
 
+class StepsNowcaster:
+    def __init__(self, precip, velocity, timesteps, **kwargs):
+        # Store inputs and optional parameters
+        self.precip = precip
+        self.velocity = velocity
+        self.timesteps = timesteps
+        self.n_ens_members = kwargs.get("n_ens_members", 24)
+        self.n_cascade_levels = kwargs.get("n_cascade_levels", 6)
+        self.precip_thr = kwargs.get("precip_thr", None)
+        self.kmperpixel = kwargs.get("kmperpixel", None)
+        self.timestep = kwargs.get("timestep", None)
+        self.extrap_method = kwargs.get("extrap_method", "semilagrangian")
+        self.decomp_method = kwargs.get("decomp_method", "fft")
+        self.bandpass_filter_method = kwargs.get("bandpass_filter_method", "gaussian")
+        self.noise_method = kwargs.get("noise_method", "nonparametric")
+        self.noise_stddev_adj = kwargs.get("noise_stddev_adj", None)
+        self.ar_order = kwargs.get("ar_order", 2)
+        self.vel_pert_method = kwargs.get("vel_pert_method", "bps")
```

Comment:
```suggestion
        self.velocity_perturbation_method = kwargs.get("velocity_perturbation_method", "bps")
```
If this works for you, too, then we should change it in the rest of the code as well.